internal/cmd/connect/command_plugin_install.go

```diff
@@ -0,0 +1,461 @@
+	if err != nil {
+		return err
+	}
+
+	pluginDir, err := getPluginDirFromFlag(cmd)
+	if err != nil {
+		return err
+	}
+
+	workerConfigs, err := getWorkerConfigsFromFlag(cmd)
+	if err != nil {
+		return err
+	}
+
+	var installation *platformInstallation
+	prompt := form.NewPrompt(os.Stdin)
+	if pluginDir == "" {
+		installation, err = getConfluentPlatformInstallation(cmd, prompt, force)
+		if err != nil {
+			return err
+		}
+		pluginDir, err = choosePluginDir(installation, prompt, force)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Check for, and possibly remove, existing installation
+	previousInstallations, err := existingPluginInstallation(pluginDir, pluginManifest)
+	if err != nil {
+		return err
+	}
+
+	if err := removePluginInstallations(previousInstallations, prompt, dryRun, force); err != nil {
+		return err
+	}
+
+	// Install
+	if err := checkLicenseAcceptance(pluginManifest, prompt, force); err != nil {
+		return err
+	}
+
+	if err := c.installPlugin(client, pluginManifest, args[0], pluginDir, dryRun); err != nil {
+		return err
+	}
+
+	// Select and update worker-configs
+	if len(workerConfigs) == 0 {
+		if installation == nil {
+			installation, err = getConfluentPlatformInstallation(cmd, prompt, force)
+			if err != nil {
+				return err
+			}
+		}
+		workerConfigs, err = chooseWorkerConfigs(cmd, installation, prompt, force)
+		if err != nil {
+			return err
+		}
+	}
+
+	if err := updateWorkerConfigs(pluginDir, workerConfigs, dryRun); err != nil {
+		return err
+	}
+
+	successStr := fmt.Sprintf("Installed %s %s.\n", pluginManifest.Title, pluginManifest.Version)
+	if dryRun {
+		successStr = addDryRunPrefix(successStr)
+	}
+	output.Printf("\n%s", successStr)
+
+	return nil
+}
+
+func parsePluginId(plugin string) (string, string, string, error) {
+	parsePluginErrorMsg := "plugin not found"
+	parsePluginSuggestions := "Provide a path to a local file or provide a plugin ID from Confluent Hub with the format: `<owner>/<name>:<version>`."
+
+	ownerNameSplit := strings.Split(plugin, "/")
+	if len(ownerNameSplit) != 2 || ownerNameSplit[0] == "" {
+		return "", "", "", errors.NewErrorWithSuggestions(parsePluginErrorMsg, parsePluginSuggestions)
+	}
+	nameVersionSplit := strings.Split(ownerNameSplit[1], ":")
+	if len(nameVersionSplit) != 2 || nameVersionSplit[0] == "" || nameVersionSplit[1] == "" {
+		return "", "", "", errors.NewErrorWithSuggestions(parsePluginErrorMsg, parsePluginSuggestions)
+	}
+
+	return ownerNameSplit[0], nameVersionSplit[0], nameVersionSplit[1], nil
+}
+
+func (c *pluginCommand) getManifest(client *hub.Client, id string) (*cpstructs.Manifest, error) {
+	if utils.DoesPathExist(id) {
+		// if installing plugin from local archive
+		return getLocalManifest(id)
+	} else {
+		// if installing plugin from Confluent Hub
+		owner, name, version, err := parsePluginId(id)
+		if err != nil {
+			return nil, err
+		}
+
+		return client.GetRemoteManifest(owner, name, version)
+	}
```

Comment:
The first branch always returns so we can save some indentation.
```suggestion
	}
	
	// if installing plugin from Confluent Hub
	owner, name, version, err := parsePluginId(id)
	if err != nil {
		return nil, err
	}
	
	return client.GetRemoteManifest(owner, name, version)
```